etc/openqa/openqa.ini

```diff
@@ -88,6 +88,18 @@
 # disable auditing of chatty events by default
 blacklist = job_grab job_done
 
+# Sets the storage duration in days for the different audit event types
+[audit/storage_duration]
+#startup = 10
+#jobgroup = 365
+#jobtemplate = 365
+#table = 365
+#iso = 60
+#user = 60
+#asset = 30
+#needle = 30
+#other = 15
```

Comment:
These values in comments suggest that they represent the defaults. However according to the documentation, and as per the code, by default nothing is cleaned up.

If this is meant as an example, I'd suggest a more verbose comment like

    # By default cleanup is disabled, see http://open.qa/docs/#_auditing_tracking_openqa_changes
    # The following categories with example values can be uncommented as needed

Reply:
Makes sense, I'll change it.